### PR TITLE
 loops-challenge completed

### PR DESCRIPTION
**Pull request by RogueStorm7**
This took me way longer than I thought it would but figured it out in a way that I can understand the functions and methods used. Please take a look and provide feedback if needed.

### CODE REVIEW

File: Mianta/Feb-15-Mianta/loops_challenge.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+print(temperature) #prints temp in a string
+    
+if temperature < 50:
+        print('Put on a jacket!')# If the temperature is below 50, tell the user to put on a jacket
+elif temperature >= 50 and temperature <= 65:
+        print('Put on a sweater!')#if the temperature is between 50 and 65, tell the user to put on a sweater.
+else:
+        print('Put on sunscreen!')#Finally, if the temperature is above 65, tell the user to put on some sunscreen.
+
```
Comment (waldenwang9966) [inline Mianta/Feb-15-Mianta/loops_challenge.py:75]: I think what the prompt really wants is to print out the clothings or sunscreen for each loop, so the if else statement should be inside the for day in days loop.

**Review by waldenwang9966 — CHANGES_REQUESTED** [review]

File: Mianta/Feb-15-Mianta/loops_challenge.py (hunk 1 above)
Comment (RogueStorm7) [reply Mianta/Feb-15-Mianta/loops_challenge.py:75]: made changes and doing a pull request now

Comment (waldenwang9966) [reply Mianta/Feb-15-Mianta/loops_challenge.py:75]: Hmmm. It's so weird. I don't see the changes you made. The files you pushed seem to be the same as the ones you pushed before.

**Comment by RogueStorm7** [comment]
I'll push it again. Idk why it didn't work but will figure it out and fix it. Thanks for your patience.

**Pull request closed without merging**